src/options.js

```diff
@@ -39,15 +41,18 @@ export function getRealmOptions({
   debugNames = false,
   uniqueSuffix,
   timeout,
+  residual,
+  serialize,
```

Comment:
I meant to put these in the argument destructuring position:

```js
export function getRealmOptions({
   compatibility = "browser",
   mathRandomSeed,
   debugNames = false,
   uniqueSuffix,
   timeout,
   residual = false,
   serialize = !residual,
   strictlyMonotonicDateNow
  }: Options): RealmOptions {
  ...
}
```

Setting these on `defaultOptions` isn't enough because that doesn't set the defaults if a custom object is provided. E.g. `getRealmOptions({})` would yield `{ ... serialize: undefined, residual: undefined }`.

These are destructuring patterns with default values. So they can be overridden if the user specifies something specific.

 `getRealmOptions({residual:true, serialize:true})` would yield `{ ... serialize: true, residual: true }`

 `getRealmOptions({residual:true})` would yield `{ ... serialize: false, residual: true }`